beam_sink/sinks/mysql.py

```diff
@@ -96,6 +116,8 @@ def finish_bundle(self):
 
 
 class _Insert(_PutFn):
+    """An internal DoFn to be used in a PTransform. Not for external use
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'e')](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)